.pipelines/onebranch/templates/template-buildrp-buildev2.yml

```diff
@@ -1,3 +1,5 @@
+parameters:
+  generationType: ''
```

Comment:
Why is this pipeline template being overloaded?  It's similar in pattern but the pattern is very different.  Name of the file does not indicate it might be used for mirroring.  Unclear what acceptable parameters are without reading the template.

Reply:
Thanks for the feedback.  I'll abandon this PR as we're not ready for a review.  I was thinking that a draft PR would not alert anyone until we're ready, but also allow CF internally to do some collaboration beforehand.